src/app/helpers/logger.node.test.js

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable global-require */
+const fs = require('fs');
+const path = require('path');
+
+let winston;
+
+describe('Logger node - for the server', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const deleteFile = (logPath, logFile) => {
+    fs.unlink(path.join(logPath, logFile), () => {});
+  };
+
+  const deleteFolder = logPath => {
+    if (fs.existsSync(logPath)) {
+      deleteFile(logPath, 'app.log');
+      fs.rmdir(logPath, err => {
+        if (err) throw err;
+      });
+    }
+  };
+
+  describe('Setting up logger', () => {
+    beforeEach(() => {
+      jest.resetModules();
+    });
+
+    describe('Folder creation', () => {
+      const logPath = path.join(__dirname, '../../..', 'log-temp');
+      const defaultLogPath = path.join(__dirname, '../../..', 'log');
+
+      beforeEach(() => {
+        process.env.LOG_DIR = logPath;
+        deleteFolder(logPath);
+        deleteFolder(defaultLogPath);
+        jest.resetModules();
+        jest.resetAllMocks();
+      });
+
+      afterAll(() => {
+        deleteFolder(logPath);
+        deleteFolder(defaultLogPath);
+      });
+
+      it('creates folder log-temp', () => {
+        require('./logger.node');
+
+        expect(fs.existsSync(logPath)).toBe(true);
+      });
+
+      it('creates default folder log when LOG_DIR isnt set', () => {
+        delete process.env.LOG_DIR;
+        require('./logger.node');
+
+        expect(fs.existsSync(defaultLogPath)).toBe(true);
+      });
+
+      it('does not create folder log-temp when it already exists', () => {
+        fs.mkdirSync(logPath);
+        jest.mock('fs');
+        fs.mkdirSync = jest.fn();
+
+        require('./logger.node');
+
+        expect(fs.mkdirSync).not.toHaveBeenCalled();
+      });
+    });
+
+    describe('Configuring Winston', () => {
+      beforeEach(() => {
+        jest.mock('winston', () => jest.fn());
```

Comment:
Not required as a change but this can be formatted slightly nicer similar to https://github.com/bbc/simorgh/blob/a1e230ccfcb0594688c5aa64f820ad1e3a81e148/src/server/index.test.js#L21-L26